Dockerfile.node7

```diff
@@ -0,0 +1,14 @@
+# Dockerfile for using Yarn along with Node.js 7.x
+# docker build -t yarnpkg/node-yarn:latest -f Dockerfile.node7 .
```

Comment:
If you do `--squash` as well, there is no need for chained commands, the resulting image is just a single layer on top of `FROM`